backend/controllers/social.controller.ts

```diff
@@ -0,0 +1,46 @@
+import { Request, Response } from 'express';
+import socialModel, { ISocial } from '../models/social.model';
+
+export const getAllSocials = async (_req: Request, res: Response) => {
```

Comment:
If whole class is related to Social, we dont need this name in this function 
getAllSocials -> getAll